examples/SendContext/SendContext.cpp

```diff
@@ -7,13 +8,16 @@
 #include "Ubidots.h"
 
 /****************************************
- * Define Constants
+ * Define Instances and Constants
  ****************************************/
 
-#ifndef UBIDOTS_TOKEN
-#define UBIDOTS_TOKEN ""  // Put here your Ubidots TOKEN
+#ifndef TOKEN
```

Comment:
This must be UBIDOTS_TOKEN  to avoid re-definitions of other instances in other libraries